bookkeeper-server/pom.xml

```diff
@@ -84,6 +84,11 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
Is it possibile not to add dependency to commons-lang3 ? it is used only for org.apache.commons.lang3.tuple.Pair which can be simply substituted by Map.Entry or other custom implmentation.